### PR TITLE
Add event driven SMS

### PR DESCRIPTION
**Pull request by jkempster34**
### Trello card
https://trello.com/c/Vt2y63Sd

### Context
We want to alert the candidates by SMS when their placement request has been accepted, or it has been accepted then cancelled.

### Changes proposed in this pull request
- Remove unused SMS templates - Left these in by accident. They were used for testing in the previous PR
- Notify the candidate by SMS when the school cancels
- Notify the candidate by SMS when the school accepts a placement request
- Update the help text above the phone number input
- Update the confirmation screens to let the school know a text has been sent along with the email.

### Guidance to review
To test need to:
1. Be running the app locally, the preview app doesn't use Notify.
2. Create a new placement request.
3. Log in as the school
4. Accept the request. SMS should be sent
5. Visit the Manage upcoming bookings section and cancel the request. SMS should be sent.

To do: 
1. ~~Improve the help text above the phone number input?~~ ✅ 
Before:
image
After:
image

2. ~~Add some text to explain to the schools that they are also sending a text? These are sent when the school clicks `Send confirmation email` in both cases. There is a confirmation page per email sent too (I haven't actually updated the preview or confirmation screens yet, just included screenshots below for clarity).~~ ✅ 

| Preview | Confirmation | SMS |
| ------- | --------- | --------- | 
| image | image | image | 
image | image | image |

After: 
image
image
image

### CODE REVIEW

**Comment by B-Bonner** [comment]
Suggested content for the Confirm your details page @jkempster34 Let me know what you think:


Screenshot 2021-12-13 at 09 58 33

**Comment by B-Bonner** [comment]
Content suggestion for the Confirmation page:

Screenshot 2021-12-13 at 10 00 49

**Comment by B-Bonner** [comment]
Content suggestion for Cancellation page:

Screenshot 2021-12-13 at 10 11 00

**Comment by jkempster34** [comment]
@B-Bonner Just done a run-through and the texts look good to me! 
image

**Comment by B-Bonner** [comment]
Here are the example messages using the longest school name:

Screenshot 2021-12-13 at 13 17 52

**Comment by jkempster34** [comment]
> > @B-Bonner Just done a run-through and the texts look good to me! image
> 
> Great. We just need to remove 'folder' from the text. As when I tested the template with the longest school name, it ran over 2 messages.

Done 👍

File: app/controllers/schools/placement_requests/acceptance/preview_confirmation_email_controller.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ def update
           @booking = @placement_request.booking
           @booking.assign_attributes(booking_params)
 
-          if @booking.valid?(:acceptance_email_preview) && @booking.accept! && candidate_booking_notification(@booking).despatch_later!
+          if @booking.valid?(:acceptance_email_preview) && @booking.accept! && candidate_booking_notifications(@booking)
```
Comment (ethax-ross) [inline app/controllers/schools/placement_requests/acceptance/preview_confirmation_email_controller.rb:19]: I find this a bit unintuitive to have something in the conditional being an action that sends an SMS; is it worth calling out separately?

**Review by ethax-ross — APPROVED** [review]
Looks good and works well - one take it or leave it comment.

**Review by leoapost — APPROVED** [review]

**Pull request merged**